xradar/accessors.py

```diff
@@ -165,6 +173,13 @@ def get_crs(self):
         radar = self.xarray_obj
         return radar.pipe(get_crs)
 
+    def apply(self, func, *args, **kwargs):
+        """
+        Apply a function to the Dataset object.
+        """
+        ds = self.xarray_obj
+        return func(ds, *args, **kwargs)
```

Comment:
We might use the included boilerplate checks within `.pipe`.

```suggestion
        return da.pipe(func, *args, **kwargs)
```

Reply:
Done!